src/store/TabsContext.tsx

```
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Tab } from '../types/Tab';
+
+type TabsContextType = {
+  tabs: Tab[];
+};
+
+export const TabsContext = React.createContext<TabsContextType>({ tabs: [] });
```

Optional: the context default value is `{ tabs: [] }`. This is acceptable, but if you want stricter safety you can type the context as `TabsContextType | undefined` and set default to `undefined` so consumers will fail fast when used outside a provider. This is a design choice rather than a requirement.